chanzuckerberg/idseq-web
# Fix some issues in analytics

# Description

This fixes some issues in analytics.

1. provide more context to GA
2. fix `request` arg in ruby
3. remove old events covered by auto events in ruby
4. add name scheme validation

# Test

1. Check new context in segment debugger
2. see no errors in ruby log
3. try valid and invalid names such as "AccessionViz_more-reads-link_clicked" and "AccessionViz_more-reads_link_clicked" and see appropriate errors
